cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/replication/ReplicatorBuilder.java

```diff
@@ -301,10 +310,30 @@ public Pull filter(PullFilter pullPullFilter) {
             return this;
         }
 
+        /**
+         * <p>Provides the mango query selector for filter criteria to be used
+         * when a pull replication calls the source database's {@code _changes} feed.
+         * </p>
+         *
```

Comment:
An additional comment should be added in here to point out that this method is only valid for replications against CouchDB 2.x APIs since the `_selector` is not available for `_changes` in CouchDB 1.x.